python-sdk/nuscenes/eval/common/loaders.py

```diff
@@ -216,8 +216,11 @@ def filter_eval_boxes(nusc: NuScenes,
     """
     # Retrieve box type.
     assert len(eval_boxes.boxes) > 0
-    first_key = list(eval_boxes.boxes.keys())[0]
-    box = eval_boxes.boxes[first_key][0]
+    box = None
+    for val in eval_boxes.boxes.values():
```

Comment:
Perhaps add a comment explaining what you are doing here? It wasn't obvious to me at first glance. Perhaps even put in a helper-method "get_one_box"? That way we can test it also ;)

Reply:
I wrapped the entire thing into a function `_get_box_class_field` and added a test. Please feel free to re-approve.